index.d.ts

```diff
@@ -37,29 +37,22 @@ declare module '@primer/components' {
       StyledSystem.BottomProps,
       StyledSystem.LeftProps {}
 
-  interface FlexItemProps
+  export interface BoxProps
     extends BaseProps,
       CommonProps,
       LayoutProps,
-      StyledSystem.FlexProps,
-      StyledSystem.JustifySelfProps,
-      StyledSystem.AlignSelfProps,
```

Comment:
Did these get removed because they already exist in `FlexProps`? 🤔 I wonder if they were included for a particular reason? Where you able to test the type changes in a TS environment? If not I can pull it into my code sandbox testing instance

Reply:
I removed these to replace them with `StyledSystem.FlexboxProps`, as this list was missing 9 sets of flexbox-based interfaces. We include the props in `FLEX` but not all those props exist in the old interface list. Here's the full list:

```typescript
export interface FlexboxProps
    extends AlignItemsProps,
        AlignContentProps,
        JustifyItemsProps,
        JustifyContentProps,
        FlexWrapProps,
        FlexDirectionProps,
        FlexProps,
        FlexGrowProps,
        FlexShrinkProps,
        FlexBasisProps,
        JustifySelfProps,
        AlignSelfProps,
        OrderProps {}
```